dev/io.openliberty.com.sun.xml.messaging.saaj/saaj.2.0.bnd

```diff
@@ -1,15 +1,27 @@
--include= ~../cnf/resources/bnd/rejar.props
+#*******************************************************************************
+# Copyright (c) 2020,2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props, \
+          ~../cnf/resources/bnd/app-resources.bnd
```

Comment:
bundle.props is already in the parent bnd.bnd file.  Not sure why here as well.  app-resources doesn't make sense since you already are declaring a ResourceProvider Service-Component.  Not sure if this is causing two to get created which would be wrong if it does.

This file name should be 2.0.bnd to be completely accurate from a bnd perspective.  sub bundle symbolic names should be the parent bnd.bnd project name then a period and then the sub bnd file name.  The same goes for the saaj.3.0.bnd file name.